generator-scrivito/app/index.js

```diff
@@ -0,0 +1,172 @@
+const Generator = require("yeoman-generator");
+
+module.exports = class extends Generator {
+  constructor(args, opts) {
+    super(args, opts);
+    this.log("Welcome in scrivito Obj/Widgets generator!");
+    this.option("obj", { type: String });
+    this.option("widget", { type: String });
+  }
+
+  start() {
```

Comment:
The `app/index.js` is only the place holder for the initial `yo scrivito` command. So it should be really simple:

```js
  start() {
    this.prompt({
      type: "list",
      name: "type",
      message: "What do you want to generate?",
      choices: ["Generate a new Obj", "Generate a new Widget"],
    }).then(answers => {
      if (answers.type === "Generate a new Widget") {
        this.composeWith("scrivito:widget");
      }
      if (answers.type === "Generate a new Obj") {
        this.composeWith("scrivito:obj");
      }
    });
  }
```

Please remove all other functions from this generator besides the `constructor` and `start`. Also all files and folders in `generator-scrivito/app/templates/` can be removed.